### ISSUE TITLE
pasting does not work only on Edge v42 (Unexpected call to method or property access.)

### ISSUE THREAD

**Issue by anki247** [ckeditor/ckeditor-dev#2303]
## Type of report

Bug - only on egde v42 (what else?)
Actually it works on v41 😕 

## Provide detailed reproduction steps (if any)

1. very first step, try to paste some text

On console it prints `SCRIPT65535: Unexpected call to method or property access.
ckeditor.js (726,445)` And the copied text in clipboard is also gone. You have to recopy the text and on second time it works

### Expected result

Display pasted text immediately. 

### Actual result

On UI it happens nothing, repeating paste action also brings no change, since there is no text on clipboard

## Other details

* Browser: Edge
* OS: Windows
* CKEditor version: 4.10.0
* Installed CKEditor plugins: 
````javascript
plugins : {
		'autolink' : 1,
		'basicstyles' : 1,
		'clipboard' : 1,
		'enterkey' : 1,
		'entities' : 1,
		'htmlwriter' : 1,
		'indentlist' : 1,
		'link' : 1,
		'list' : 1,
		'toolbar' : 1,
		'undo' : 1,
		'wysiwygarea' : 1
	},
````
CONFIG.JS
````javscript

CKEDITOR.editorConfig = function( config ) {
    config.removeButtons = 'Cut,Copy,Paste,Anchor,Underline,Strike,Subscript,Superscript';
    config.removeDialogTabs = 'link:advanced';

    // ** CUSTOM config

    // The toolbar
    config.toolbar = [
        [ "Undo", "Redo" ], // Line break - next group will be placed in new line.
        { name: "basicstyles", items: [ "Bold", "Italic", "Underline", "NumberedList", "BulletedList" ] },
        { name: "links", items: [ "Link", "Unlink"] },
    ];
    config.basicEntities = false;
    config.entities_greek = false;
    config.entities_latin = false;
    config.entities_additional = '';
    config.magicline_color = "#fff";
    config.linkShowAdvancedTab = false;
    config.linkShowTargetTab = false;
};


CKEDITOR.on( 'instanceReady', function( ev ) {
    var editor = ev.editor;

    // config object for editor
    var disableFormattingObj = {
        indent: false,
        breakBeforeOpen: false,
        breakAfterOpen: false,
        breakBeforeClose: false,
        breakAfterClose: false
    };

    // disable any formatting
    ["p", "ol", "ul", "li", "a"].forEach(function(tag) {
        editor.dataProcessor.writer.setRules(tag, disableFormattingObj);
    });

    editor.dataProcessor.htmlFilter.addRules({
        elements : {
            a : function(element) {
                // add rel="noopener noreferrer" to link
                if (!element.attributes.rel) {
                    element.attributes.rel = "noopener noreferrer";
                }
                // add target="_blank" to link
                if (!element.attributes.target && element.attributes.target !== "_blank") {
                    element.attributes.target = "_blank";
                }
                // add class="user-link" to link
                if (!element.attributes.class) {
                    element.attributes.class = "user-link";
                }
            }
        }
    });
} );
````

**Comment by engineering-this** [comment]
Hi

We are currently working on related issues #1832 and #1943. Solution from #1850 is also solving your case.

**Comment by gingebaker** [comment]
Just tested this with Version 4.10.1. This bug seems to be fixed!

**Comment by engineering-this** [comment]
I'm closing it as I can't reproduce it any more. It seems to be fixed on Edge 17.17134

Thanks @gingebaker for information.
